build.xml

```diff
@@ -377,6 +377,16 @@
     <exec command="touch ${srcdir}/vufind-exception.log" />
     <exec command="chmod a+w ${srcdir}/vufind-exception.log" />
 
+    <!-- Update contentsecuritypolicy.ini to enforce the CSP and disable strict-dynamic that's not supported on all platforms. -->
+    <copy file="${srcdir}/config/vufind/contentsecuritypolicy.ini" tofile="${srcdir}/local/config/vufind/contentsecuritypolicy.ini">
```

Comment:
Do we always want to do this, or should there be a property that controls whether or not we run tests with CSP active? It might be useful to be able to turn it on or off if we want to verify whether a test failure is CSP-related.

Reply:
I think we always want to do this, but I'll add an option to turn it off for easier testing.